### ISSUE TITLE
I was wondering what's the meaning of averaged accuracy over three splits.

### ISSUE THREAD

**Issue by BenjaminJonghyun** [kenshohara/3D-ResNets-PyTorch#140]
Hi there!

I'm studying action recognition tasks for my research area.
As soon as I saw a lot of papers related to action recognition. Most of the papers used mean accuracy over three splits as an evaluation method. 
I really want to know the evaluation method.

In my knowledge, I guess three splits mean train set, validation set, and test set.

please tell me about the details.

**Comment by ziming-liu** [comment]
there are three different splits  in UCF, each split has train set  validation set(is also test set) . and each split have different samples for train set, val set.  in a word, they have three ways to  decide train/test

**Comment by BenjaminJonghyun** [comment]
@ziming-liu So, mean accuracy is that they calculated an average of accuracy which used three split methods for train/test. Is it right?

**Comment by slighting666** [comment]
@BenjaminJonghyun 
I think it was to figure out how accurate each split was and then take the average
